src/cursor.js

```diff
@@ -5,7 +5,7 @@ export class Cursor extends BaseCursor {
 
   writeBufferPadded(buffer) {
     let padding = calculatePadding(buffer.length);
-    let paddingBuffer = new Buffer(padding);
+    let paddingBuffer = Buffer.alloc(padding);
     paddingBuffer.fill(0);
```

Comment:
`Buffer.alloc` fills the buffer with `0`s by default so we don't need to call `fill`.